packages/icons/scripts/copy-and-optimize-to-dist.js

```diff
@@ -0,0 +1,31 @@
+const chalk = require("chalk");
+const { checkFolderExist } = require("../../../scripts/helper");
+const fs = require("fs");
+const { optimize } = require("./optimize");
+
+const SRC_PATH = "src";
+const DIST_PATH = "dist";
+
+const generateSvg = (svgList, dir) => {
```

Comment:
Are you really "generating" an svg? It seems more like create (or writing) a file.